<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Table</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #252525;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>135</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #217245; 
color: white;</string>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWindow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>675</width>
      <height>370</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>530</width>
      <height>370</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>530</width>
      <height>370</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView {
    background-color: #353535;
	color: white;
}

QHeaderView::section{
	background-color: #353535;
	color: white;
}

QTableView QTableCornerButton::section {
    background: #353535;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="regionInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #353535;
border: 1px solid white;
border-radius: 5px;
color: white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="columnInput">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>450</y>
      <width>135</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #353535;
border: 1px solid white;
border-radius: 5px;
color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>450</y>
      <width>135</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #217245; 
color: white;</string>
    </property>
    <property name="text">
     <string>Load data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calcButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>450</y>
      <width>135</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #217245; 
color: white;</string>
    </property>
    <property name="text">
     <string>Calculate metrics</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>675</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #353535;
padding-left: 10px;
color: white;
</string>
    </property>
    <property name="text">
     <string>RESULT:</string>
    </property>
   </widget>
   <widget class="QLabel" name="FilePathLabel">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>10</y>
      <width>530</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #353535;
padding-left: 10px;
color: white;</string>
    </property>
    <property name="text">
     <string>FILE SOURCE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
